### DISCUSSION TITLE
Will i be able to make mods or only scripts using this?

### DISCUSSION THREAD

**Discussion by cad3264** [moongardenmods/allium#40] [Q&A]
Is it possible i could make a JAR file that includes some sort of lua scripts using this, or will i only be limited to putting scripts in the allium folder?

**Comment by hugeblank** [comment]
The short answer: yes, you can already do this
The long answer:
Allium will be able to run scripts that are:
- standalone: provided as a zip in the `allium` directory, as you say
- within a mod container: through the `resources/allium` directory within a jar file (iirc)

The goal was to allow for developers to pick what degree they want their mod to be extensible through Allium.
